recipes-bsp/tegra-binaries/tegra-configs-alsa_%.bbappend

```diff
@@ -0,0 +1,12 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+
+SRC_URI:append = "\
+    file://asound-xavier-nx.conf \
+"
+
+python() {
+    machine = d.getVar('MACHINE')
+
+    if machine == "jetson-xavier-nx-devkit" or machine == "jetson-xavier-nx-devkit-emmc":
+        d.setVar("TEGRA_AUDIO_CONFIG", d.getVar('WORKDIR') + "/asound-xavier-nx.conf")
```

Comment:
Is there a way to make this a weak assignment, so that other appends (in a custom distro layer that includes meta-tegra-community for other reasons) can override this behavior?

Reply:
Yeah, the best that I could think of is.

```python
def do_select_alsa_config(d):
    machine = d.getVar('MACHINE')

    if machine == "jetson-xavier-nx-devkit" or machine == "jetson-xavier-nx-devkit-emmc":
        return d.getVar('WORKDIR') + "/asound-xavier-nx.conf"

TEGRA_AUDIO_CONFIG ?= "${@do_select_alsa_config(d)}"
```
From what I am seeing though it still uses the stock version set in meta-tegra/master. May need to add another commit in master with TEGRA_AUDIO_CONFIG ??= .. or let people know of the names of potential alsa configs they can include.

Reply:
Yes, that overrides it. Made change.